### ISSUE TITLE
How to use it with Postman

### ISSUE THREAD

**Issue by vuminhquang** [webmotions/fake-authentication-jwtbearer#9]
Hi,

Thank you for your useful library.
I've setup the API server using WebMotions.Fake.Authentication.JwtBearer.
Now I want to call my API from Postman, with the "Authorization" Header like this string:
"Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJiYzI5Y2ZhMC0zZDA5LTRkOGMtOTQ4MS0yZThjMGViMWFhNmMiLCJuYW1lIjoiSm9obiBEb2UiLCJpYXQiOjE1MTYyMzkwMjJ9.QeUAvDvmNlCHoq1LugpJglaHb7HkqzEBDsxH-hXfa5A"

Payload for this Token is
{
  "sub": "bc29cfa0-3d09-4d8c-9481-2e8c0eb1aa6c",
  "name": "John Doe",
  "iat": 1516239022
}

But I get an Unauthorized response.

Do you have any clue on calling from Postman?
Thank you.

**Comment by DOMZE** [comment]
Hi @vuminhquang,

This library was made with the intention of using it in a testing fashion. That is, using TestServer in ASP.NET Core.

It is not made to be used as a "real" library for authentication. However, if you wish to do so (for demo purposes), just following add the library to your Authentication Pipeline.

Make sure to set the FakeJwtBearerBearerValueType in the options to Jwt.

**Comment by vuminhquang** [comment]
Hi @DOMZE,
Yes, we are using for demonstration, mimic AZ B2C.

FakeJwtBearerBearerValueType in the options to Jwt was added.
![image](https://user-images.githubusercontent.com/20512463/149264358-011005ac-1344-49fc-8464-28513404c762.png)

Postman headers like this
![image](https://user-images.githubusercontent.com/20512463/149264540-a3eefcda-7094-4c73-a88f-e014a99f3fe4.png)

We've got this in the console
![image](https://user-images.githubusercontent.com/20512463/149264660-4a2b28b3-3221-45f1-a72c-009fcd86542a.png)

The result still 401

Thank you.
